### ISSUE TITLE
Missing planets/moons in bookmarks - possible EDSM sync issue

### ISSUE THREAD

**Issue by goscickiw** [EDDiscovery/EDDiscovery#2045]
I tried to bookmark the Thargoid wreck locations, but some of the planetary bodies in HIP 17862 and HIP 17403 are missing from the drop-down menu in the "Surface Bookmarks In System" section.
I already deleted and re-downloaded the system databases, but it didn't fix the issue.

For example, HIP 17862 6 c a:
![2](https://user-images.githubusercontent.com/39399945/42094382-ea9f55a6-7baf-11e8-915f-7d4d4fb6419b.PNG)

It is in the EDSM database, marked as landable, with materials listed:
![4](https://user-images.githubusercontent.com/39399945/42094212-4a1803c6-7baf-11e8-8707-3de7f1f9401c.PNG)

However, in the EDDiscovery Scan section it appears as non-landable, with no materials listed:
![3](https://user-images.githubusercontent.com/39399945/42094342-c9b39b4a-7baf-11e8-96a0-cf985d51475b.PNG)

When I land on it and try to make a bookmark by clicking the green button on the compass, it shows this:
![1](https://user-images.githubusercontent.com/39399945/42094831-40f5775e-7bb1-11e8-851e-250cc5450672.PNG)

**Comment by robbyxp1** [comment]
Will investigate.

**Comment by klightspeed** [comment]
With the moon, did you have a Detailed Surface Scanner installed when you scanned it?  Local scans override EDSM data, and without a DSS, no material data would have been gathered.

**Comment by goscickiw** [comment]
Before, I have considered that not using DSS might have caused the issue, but I also thought that ordinary discovery scan included the information about landability, and it's probably not that. But now that you said that this really may be the cause, I took the DSS and flew back there. I scanned the moon, landed on it... and now it works. I can create the bookmark by using the button on the compass, and the moon is visible on the planetary bodies list.
Usually, when I try to use the simplest solutions, they don't work. So I start to omit them. When I omit them, they later turn out to be the ones that solve the problem.

I think there should be a warning message that a detailed surface scan needs to be done (if it hasn't been), when attempting to create the bookmark with the compass feature. Then the DataError issue would be solved, and users would know what to do instead of reporting bugs like I did.

Also, I think that the program features like bookmarks should not use player scan data over EDSM data, if the EDSM data is more detailed. If the player scan data is necessary, there could be a separate database for it. If a player scan from that database becomes as detailed or more detailed than the EDSM data, then it will be merged into the EDSM database.
But I don't know if changing the program so much is possible at this point.
